README.md

```diff
@@ -1,42 +1,37 @@
-# powerline-web-fonts
+# Powerline Web Fonts
 
 [**Powerline** Web Fonts](https://github.com/powerline/fonts) working in [Secure Shell](https://chrome.google.com/webstore/detail/secure-shell/pnhechapfaindjhompbnflcldabbghjo) and working also on **Chromebook** (or pretty much anything running Chrome).
 
-There are multiple `font-family` to choose from (all Powerline-enabled):
-
-  * `Anonymous Pro` - [Anonymice
-    Powerline](https://github.com/powerline/fonts/tree/master/AnonymousPro)
-  * `DejaVu Sans Mono` -
-    [DejaVu Sans Mono for Powerline](https://github.com/powerline/fonts/tree/master/DejaVuSansMono)
-  * `Hack` - [Hack Webfont](https://github.com/chrissimpkins/Hack)
-  * `Inconsolata` - [Inconsolata for
-    Powerline](https://github.com/powerline/fonts/tree/master/Inconsolata)
-  * `Inconsolata-g` - [Inconsolata-g for
-    Powerline](https://github.com/powerline/fonts/tree/master/Inconsolata-g)
-  * `Iosevka` - [Iosevka Webfont](https://github.com/be5invis/Iosevka)
-  * `Liberation Mono` - [Literation Mono
-    Powerline](https://github.com/powerline/fonts/tree/master/LiberationMono)
-  * `PT Mono` - PT Mono for Powerline
-  * `Source Code Pro` - [Source Code for Powerline](https://github.com/powerline/fonts/tree/master/SourceCodePro)
-  * `Ubuntu Mono` - [Ubuntu Mono derivative
-    Powerline](https://github.com/powerline/fonts/tree/master/UbuntuMono)
-
-See [preview.html](https://rawgit.com/wernight/powerline-web-fonts/master/preview.html) and [Slant](http://www.slant.co/topics/67/~programming-fonts) if you don't know which to choose.
+There are several Powerline-enabled typefaces to choose from:
+
+  * **Anonymous Pro**  [Anonymice Powerline](https://github.com/powerline/fonts/tree/master/AnonymousPro)
```

Comment:
I kept them as code because you need to copy/paste that specific string. Could be **`...`**.